src/v/kafka/requests/fetch_request.cc

```diff
@@ -315,7 +315,7 @@ static ss::future<fetch_response::partition_response> read_from_partition(
                 };
             });
       })
-      .handle_exception_type([](const ss::timed_out_error&) {
+      .handle_exception_type([](const raft::offset_monitor::wait_aborted&) {
```

Comment:
do you want both, or is this a bug?

Reply:
Only `raft::offset_monitor::wait_aborted` is relevant.